src/app.js

```diff
@@ -47,6 +51,9 @@ app.use("/cart",
   })
 );
 
+const orderRoutes = require('./routes/order.routes');
```

Comment:
I suggest you put this route in the designated `routes` folder instead of the `app.js`